.github/workflows/codeql-analysis.yml

```diff
@@ -4,15 +4,13 @@ on:
   push:
     branches: [ master ]
     # only run if files in src/ have been changed
-    paths:
-      - 'src/**'
+    paths-ignore:
+    - '**/*.md'
```

Comment:
@Noordfrees do you think just ignoring md files would be ok?

Reply:
Hm, our current policy is to always run the full CI, though branches that can't possibly have an effect on the CI are sometimes merged without waiting for required checks. This also helps a bit with pointing out rare race conditions in the testsuite. So please remove all `paths-ignore` statements so the full CI is run regardless of whether it is needed. Other than than LGTM :)